server.go

```diff
@@ -176,23 +171,20 @@ func (s *Server) Boot() error {
 func (s *Server) goAcceptConnection(listener net.Listener) {
 	s.wait.Add(1)
 	go func(listener net.Listener) {
-	loop:
+		defer s.wait.Done()
```

Comment:
this was just legit missing.  We were hanging forever because they decremented the wait group lol